src/toggle-class.js

```diff
@@ -1,11 +1,27 @@
 
+// This global variable is required so that the body class is removed only
```

Comment:
Why do this here? Whats preventing you of tracking the count of opened modals in the call site?